documentatie/gebruikersdocumentatie/installeren-en-starten.md

```diff
@@ -1,17 +1,56 @@
-# Abacus installeren en starten
+# Abacus testen, installeren en starten
 
-Het installeren en starten van Abacus kan op verschillende manieren, met elk verschillende doeleinden.
+Het testen, installeren en starten van Abacus kan op verschillende manieren, met elk verschillende doeleinden.
 Hieronder vind je instructies voor de verschillende methodes.
 
-## Linux en macOS
+## Testen
```

Comment:
Als we het over testen gaan hebben, dan moeten we ook ergens de gautomtatiseerde tests vermelden.

Moet misschien niet meer zijn dan: als je onze geautomatiseerde tests wil draaien, moet je de repo klonen of downloaden en dan in de READMEs van de frontend en backend kijken. En misschien moet deze "Testen" sectie voor geautomatiseerde tests alleen verwijzen naar "Methode 4: Handmatig starten" en zetten we daar een paar zinnen neer.

Kortom, vind vooral dat we het ergens moeten vermelden, want de mensen die de tests willen draaien, zouden dat zonder hulp moeten kunnen, want die klonen de repo, kijken naar de READMEs, etc.

Reply:
Misschien dit kopje 'Uitproberen' noemen?